openpgp/query.go

```diff
@@ -0,0 +1,134 @@
+/*
+   Hockeypuck - OpenPGP key server
+   Copyright (C) 2012-2014  Casey Marshall
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Affero General Public License as published by
+   the Free Software Foundation, version 3.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Affero General Public License for more details.
+
+   You should have received a copy of the GNU Affero General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package openpgp
+
+import (
+	"database/sql"
+
+	"github.com/jmoiron/sqlx"
+)
+
+type query interface {
```

Comment:
Good point, my initial thought was to just push this into one file to have a large dump of all the different sql queries. I definitely see the benefit of seperating this out. Ideally I will be able to get it to a point where it doesn't have to live in the openpgp package.